pkgs/top-level/all-packages.nix

```diff
@@ -10236,6 +10236,21 @@ with pkgs;
   apacheHttpd_2_4 = callPackage ../servers/http/apache-httpd/2.4.nix { };
   apacheHttpd = apacheHttpd_2_4;
 
+  /**
+    Create an attribute set of packages built for the specified apache-httpd package.
+    It contains apache-httpd modules such as `auth_mellon`, `ca`, and `crl`.
+
+    # Parameters (positional)
```

Comment:
I think we use 'inputs' by convention. Is there a special reason to use a different term?